src/web/src/components/Schedule.vue

```diff
@@ -81,8 +77,8 @@ export default {
     ScheduleEvent: ScheduleEventComponent,
   },
   props: {
-    schedule: {
-      default: () => new Schedule(),
+    possibility: {
```

Comment:
why are we renaming this to `possibility`? a possibility is a schedule but a schedule is not necessarily a possibility